source/pom.xml

```diff
@@ -69,7 +69,6 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
-                    <encoding>${source.encoding}</encoding>
```

Comment:
Okay, do you have a link to the relevant part of the documentation? I don't really have much experience with Maven, and I'm curious. All I could find is this: https://maven.apache.org/plugins/maven-resources-plugin/examples/encoding.html (thus my initial question)